frontend/packages/console-app/src/components/guided-tours/utils/guided-tour-utils.ts

```diff
@@ -15,3 +15,7 @@ export const getGuidedToursWithStatus = (): GuidedTourCatalogItem[] => {
     };
   });
 };
+
+export const getGuidedTour = (tourID: string): GuidedTourItem => {
+  return tourID && mockGuidedTours.find((tour) => tour.id === tourID);
```

Comment:
You don't want to return a boolean:
```suggestion
  return mockGuidedTours.find((tour) => tour.id === tourID);
```

Reply:
updated the util to return undefined as returned by Array.find